README docs

gabfr opened the pull request:
Hi Juliano!

First of all, thanks for such a great project! I want to contribute improving it with all details I can work on.

As my first contribution I wrote a simple README on how to run the code with the `Dockerfile` and an usage example based on the one you wrote within the `public/index.html`

Suggestions nor a roadmap are welcome to guide my contributions!

Thanks again, hope you like

gabfr committed:
docs: writing a simple and straight forward README

gabfr committed:
docs: creating a usage example

retired from the `public/index.html` file

stackchain commented on README.md:
```diff
@@ -1,2 +1,36 @@
+**Thanks a lot to the original author of this project:** 
+
+ - **[Juliano Lazzarotto](https://github.com/stackchain)** ([his LinkedIn profile](https://www.linkedin.com/in/juliano-lazzarotto/)).
+
```
Hi @gabfr don't worry about mention me, I'd suggest you to add a new file, as many projects around do, **contributors.md** and add us both! And remove this section from the README.md, what do you think?

gabfr replied:
@stackchain checkout my last commit!

stackchain reviewed (changes requested):
What do you think about creating a contributors.md file?

gabfr committed:
docs: creating the `CONTRIBUTORS.md` file

Also removing the credits section from the body of the README and replacing by
contributors section with a link to the markdown file
